src/back/msl/mod.rs

```diff
@@ -135,6 +135,8 @@ pub enum Error {
     FeatureNotImplemented(String),
     #[error("module is not valid")]
     Validation,
+    #[error("BuiltIn {0:?} is not supported")]
+    UnsupportedBuiltIn(crate::BuiltIn),
```

Comment:
why are we having this error if the MSL backend just works around this builtin anyway?